home.html

```diff
@@ -0,0 +1,128 @@
+                      </ul>
+                  </li>
+                  <li class="program-list">
+                    <ul class="flex program-flex">
+                        <li><i class="fa fa-volume-up program-icon"></i></li>
+                        <li class="program-info">Lecture</li>
+                        <li class="program-talk">Listen to the speaker from various countries about the fast growing demand of web designer and developers.</li>
+                    </ul>
+                </li>
+                <li class="program-list">
+                    <ul class="flex program-flex">
+                        <li><i class="fa fa-home program-icon"></i></li>
+                        <li class="program-info">Forum</li>
+                        
+<li class="program-talk">Appreciate use of web accessibilty, web designing and app creations from art center, Lagos.</li>
+                    </ul>
+                </li>
+                <li class="program-list">
+                    <ul class="flex program-flex">
+                        <li><i class="fa fa-volume-up program-icon"></i></li>
+                        <li class="program-info">Lecture</li>
+                        <li class="program-talk">Listen to the speaker from various countries about the fast growing demand of web designer and developers.</li>
+                    </ul>
+                </li>                 
+              </ul>
+               <a href="#" class="program-anchor">join Global Tech 2021</a>
+          </section>
+          <section class="featured-section">
+              <h2 class="featured-header">Featured Speakers</h2>
+              <ul class="flex featured-ul">
+              </ul>
+          </section>
+          <section class="partner-section">
+              <h2 class="partner-header">Partner</h2>
+              <ul class="flex partner-ul">
+                  <li class="partner-list">
+                     <img class="partner-img" src="images/mozilla-1.svg" alt="sponsors">
+                  </li>
+                  <li class="partner-list"><img class="partner-img" src="./images/google-2015.svg" alt="sponsors"></li>
+                  <li class="partner-list"><img class="partner-img" src="./images/naver-2.svg" alt="sponsors"></li>
+                  <li class="partner-list"><img class="partner-img" src="./images/netflix-4.svg" alt="sponsors"></li>
+                  <li class="partner-list"><img class="partner-img" src="./images/airbnb.svg" alt="sponsors"></li>
+              </ul>
+          </section>
+      </main>
+      <footer class="footer">
+        <ul class="first-footer">
+            <li><i class="fa fa-first-order footer-icon"></i> </li>
+            <li class="copyright">Copyright 2021</li>
+        </ul>
+        <ul class="second-footer">
+            <li class="second-footer-list">2021 Global tech Rights Reserved.</li>
+        </ul>
+      </footer>
```

Comment:
- [ ] You should follow the given design template for all sections of your page. Footer doesn't look like the given design concept. Please check the template.

![Screenshot from 2021-07-01 20-19-02](https://user-images.githubusercontent.com/1642014/124164746-b707c500-daa9-11eb-85ec-acd67c2519a3.png)